src/test/scala/com/fulcrumgenomics/commons/util/StringUtilTest.scala

```diff
@@ -110,7 +110,18 @@ class StringUtilTest extends UnitSpec {
     StringUtil.addSpacesToCamelCase("b") should be ("b")
   }
 
-  it should "add a space between adjacen upper case characters" in {
```

Comment:
this was my best contribution in this PR

Reply:
😄